drud/ddev
# Add LANG default to web and db containers

## The Problem/Issue/Bug:

See https://drupal.slack.com/archives/C5TQRQZRR/p1624884000021500

If you try to use text with diacriticals inside the web or DB containers it just fails. 

For example, try to use "äöüÄÖÜßéêè" on the command line in `ddev ssh`

## How this PR Solves The Problem:

This seems to be a problem with the default LANG.

It might be better to do this in the Dockerfile build. 

## Manual Testing Instructions:

`ddev ssh` and then
`ls -l äöüÄÖÜßéêè`

or `echo äöüÄÖÜßéêè > junk.txt` - the text should end up in junk.txt.




<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3073"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
